index.html

```diff
@@ -26,8 +27,82 @@
+                    <div class="website-description__icons-item">
+                        <img src="/img/fire.svg" alt="Fast icon">
+                        <div class="website-description__icons-text">
+                            <h3 class="website-description__icons-header">Fastest possible</h3>
+                            <p class="website-description__icons-paragraph">Our algorithms are as fast as possible,
+                                there is no way someone did it better</p>
+                        </div>
+                    </div>
+                    <div class="website-description__icons-item">
+                        <img src="/img/klocki.svg" alt="Fast icon">
+                        <div class="website-description__icons-text">
+                            <h3 class="website-description__icons-header">Easy and accessible</h3>
+                            <p class="website-description__icons-paragraph">Each sorting algorithm has a clear
+                                description to help you understand the concept</p>
+                        </div>
+                    </div>
+                    <div class="website-description__icons-item">
+                        <img src="/img/mobile.svg" alt="Fast icon">
+                        <div class="website-description__icons-text">
+                            <h3 class="website-description__icons-header">Mobile friendly</h3>
+                            <p class="website-description__icons-paragraph">You can use our website comfortably on
+                                various devices, including phones and tablets</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="website-description__media-box">
+                    <p class="website-description__media-box-text">Short movie or GIF about sorting algorithms</p>
+                </div>
+            </div>
+        </section>
+        <section class="about-me">
+            <div class="wrapper">
+                <div class="about-me__container">
+                    <img src="/img/quotemark.svg" alt="Quotemark" class="about-me__quotemark">
+                    <div class="about-me__text-box">
+
+                        <p class="about-me__text">“I have learnt so much using this website! I recommend it to everyone
+                            that
+                            starts their journey with algorithms and web development!”</p>
+                        <p class="about-me__author-info">Talyah Kodumal, CTO and Co-Founder airly</p>
+                    </div>
+                    <div class="about-me__image-box">
+                        <img src="/img/author-photo.svg" alt="My photo">
+
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer class="footer wrapper">
+        <div class="footer__nav-container"> <img src="/img/footer-logo.svg" alt="Website logo" class="footer__logo">
```

Comment:
To też może być `<nav>`.
Formatowanie - `<img>` powinien znaleźć się w kolejnej linijce